DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.MyAccount
+{
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Data.Models.User;
+
+    public class EditDetailsViewModel
+    {
+        public EditDetailsViewModel() { }
```

Comment:
Do we need/use the empty constructor?

Reply:
Yes, it's needed for some model binding thing. Without it we get:
System.InvalidOperationException: Could not create an instance of type 'DigitalLearningSolutions.Web.ViewModels.MyAccount.EditDetailsViewModel'. Model bound complex types must not be abstract or value types and must have a parameterless constructor. Alternatively, give the 'model' parameter a non-null default value.